### DISCUSSION TITLE
Do 0 ao Dev

### DISCUSSION THREAD

**Discussion by Bvianaa** [community/community#142837] [Programming Help]
### Oi! Pode me ajudar?  
 
Prazer, sou estudante de analise e desenvolvimento de sistemas!
Estou louco para iniciar na área de programação e vim perguntar para vocês, por onde começar a carreira, como help desk, como analista suporte junior, estou estudando java, tenho conhecimento de Gestão de Infraestrutura, DevOps, Tecnico de Informatica. 

Quero ganhar pratica!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @Bvianaa, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
